drivers/interrupt_controller/intc_vexriscv_litex.c

```diff
@@ -103,6 +109,10 @@ static void vexriscv_litex_irq_handler(const void *device)
 		ite = &_sw_isr_table[GPIO_IRQ];
 		ite->isr(ite->arg);
 	}
+
+#ifdef CONFIG_WDT_LITEX
+	DT_FOREACH_STATUS_OKAY(litex_watchdog, IRQ_FKT);
```

Comment:
Could you apply a similar change to other IRQs handled in this routine?

Reply:
changed it to use `DT_FOREACH_STATUS_OKAY_NODE`. That way the interrupt controller doesn't has to be modified anymore, when new peripherals are added. This way we can also support custom out off tree peripherals and drivers with interrupts without having to edit this.